apps/src/applab/applab.js

```diff
@@ -1041,17 +1041,19 @@ Applab.init = function(config) {
           studioApp.resetButtonClick();
         }
       });
-      var scale = vizAppWidth / Applab.appWidth;
       var gridSize = 20;
```

Comment:
It looks like we don't expect grid size to change? If true, I'd change it to something like GRID_SIZE to be more clear that this is a constant.

It also looks like we only ever use half grid size, so you could just replace it with HALF_GRID_SIZE = 10 if you wanted.